web/app/routes/start/index.tsx

```diff
@@ -21,6 +22,12 @@ export default function StartQuizIndexRoute() {
 
     const startQuiz = async (quizId: number | undefined) => {
         // @ts-ignore
+        fetcher(`${window.env.API_URL}/quiz/${quizId}`)
```

Comment:
legg window.env greier i en egen variabel så er dere fri // @ts-ignore hele tia:)

Reply:
Hold your horses for a bit, this is a remix issue

Reply:
How long should i hold my horses @perkynades? They are getting pretty impatient 🏇 💨